templates/definition/meter/sonnenbatterie_eco56.yaml

```diff
@@ -48,5 +50,58 @@ render: |
     source: http
     uri: http://{{ .host }}:{{ .port }}/rest/devices/battery
     jq: .M30 # SOC relative to usable capacity (.M05 # display SOC)
+  batterymode:
+    # use a sequence to propagate resets to the watchdog
+    source: sequence
```

Comment:
Schau bitte in den anderen Templates- wozu die äußere Sequence?

Reply:
Damit der request der den modus ändert nur ein einziges mal ausgeführt wird und nicht alle 30s erneut. Oder habe ich da einen Denkfehler?